.github/ISSUE_TEMPLATE/workcycle_build_and_deploy.md

```diff
@@ -87,10 +68,8 @@ Shares a server with lib-jobs, no separate playbook
 ### Vue applications
 #### Allsearch Frontend
 - [ ] [Staging](https://allsearch-staging.princeton.edu/)
-  - [ ] Playbook
   - [ ] Deploy
 - [ ] [Production](https://allsearch.princeton.edu/)
-  - [ ] Playbook
   - [ ] Deploy
 #### Static Tables
 - [ ] [Staging](https://static-tables-staging.princeton.edu/)
```

Comment:
Remove the static tables staging playbook.

Reply:
I removed it. Could you re-review when you have a chance. Thanks.